src/views/PlanTargets.vue

```diff
@@ -0,0 +1,198 @@
+
+export default {
+  name: 'PlanTargets',
+  components: {
+    TargetCard,
+  },
+  data() {
+    return {
+      site_info: {},
+      easylist: list,
+      target: {},
+      targlist: '',
+      user: {
+        lat1: '',
+        lon1: '',
+        dateobs: '',
+        tzinfo: 'my',
+        offset: '',
+        lcltime: '',
+        lcloffset: '',
+      },
+    };
+  },
+  created: function() {
+    const url = "https://api.photonranch.org/api/all/config"
+    axios.get(url).then(response => {
+      for (let site in response.data) {
+        Vue.set(this.site_info, site, {
+          latitude: response.data[site].latitude,
+          longitude: response.data[site].longitude,
+          name: response.data[site].name,
+          siteoffset: response.data[site].TZ_database_name
+        })
+      }
+    })
+  },
+  methods: {
+    setLatLong() {
+      const ddl = document.getElementById('lcltime');
+      const selectedOption = ddl.options[ddl.selectedIndex];
+      this.user.lat1 = selectedOption.getAttribute('lat');
+      this.user.lon1 = selectedOption.getAttribute('lon');
+    },
+    submitForm() {
+      this.user.offset = new Date(this.user.dateobs).getTimezoneOffset();
+      this.user.lcloffset = moment.utc(this.user.dateobs).tz(this.user.lcltime).utcOffset()
+
+      var diclist = [];
+
+      if (this.user.tzinfo == 'my') {
+        var starttime=this.user.dateobs
+      } else if (this.user.tzinfo == 'utc') {
+        var starttime=moment(this.user.dateobs).subtract(this.user.offset, 'm').toDate()
+      } else if (this.user.tzinfo == 'lcl') {
+        var starttime=moment(this.user.dateobs).subtract(this.user.offset+this.user.lcloffset, 'm').toDate()
+      }
+
+      var endtime = moment(starttime).add(30, 'm').toDate();
+      for (var i = 0; i < this.easylist.length; ++i) {
+        var altstart = helpers.eq2altaztimesens(this.easylist[i].ra, this.easylist[i].dec, this.user.lat1, this.user.lon1, starttime)[0]
+        var altend = helpers.eq2altaztimesens(this.easylist[i].ra, this.easylist[i].dec, this.user.lat1, this.user.lon1, endtime)[0]
+        if (altstart>35 && altend>35) {
+          diclist.push({
+            "name": this.easylist[i].name, 
+            "type": this.easylist[i].group, 
+            "image": "/targs/DefaultTargetImages/"+this.easylist[i].name.replace(/ /g, "")+".jpg",
+            "ra": this.easylist[i].ra,
+            "dec": this.easylist[i].dec,
+            "starttime": starttime,
+            "altstart": altstart,
+            "altend": altend
+            })
+        }
+      }
+      console.log(diclist)
+      this.targlist=diclist;
+
+      if (window.screen.availWidth<970) {
+        setTimeout(function(){ document.getElementById("results").scrollIntoView({behavior: 'smooth'}); }, 30);}
+    },
+  },
+};
+</script>
+
+<style lang="css" scoped>
+  .{targlist {
+    margin-top: 100px;
+  }}
+  .columns {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content:center;
+  }
```

Comment:
I think the `columns` class is already defined by Bulma. It's generally preferable to add your own custom class for extra styles rather than overload an existing class.